.release/gitbook-release.sh

```diff
@@ -105,6 +106,7 @@ git_clean() {
   git clean -fd
   git remote rm doc-site
   echo "Checking out to $BRANCH"
+  git reset --hard origin/$BRANCH
   git checkout $BRANCH
```

Comment:
Is the checkout needed after a hard reset? Should we be doing the reset after the checkout? It probably don't matter here but I'm curious why we the hard reset.